tests/compile/test_sequence_parallelism.py

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import tempfile
+from pathlib import Path
+
+import torch
+
+from vllm import LLM, SamplingParams
+from vllm.config import CompilationConfig
+
+ALL_REDUCE_OP = "torch.ops.vllm.all_reduce.default"
+ALL_GATHER_OP = "torch.ops.vllm.all_gather.default"
+REDUCE_SCATTER_OP = "torch.ops.vllm.reduce_scatter.default"
+
+
+def count_comm_ops(graph_path):
+    all_reduce_cnt = 0
+    all_gather_cnt = 0
+    reduce_scatter_cnt = 0
+    try:
+        with open(graph_path) as f:
+            for line in f:
+                if ALL_REDUCE_OP in line:
+                    all_reduce_cnt += 1
+                if ALL_GATHER_OP in line:
+                    all_gather_cnt += 1
+                if REDUCE_SCATTER_OP in line:
+                    reduce_scatter_cnt += 1
+    except FileNotFoundError:
+        print(f"Error: File '{graph_path}' not found.")
+    except Exception as e:
+        print(f"Error: {e}")
+    return all_reduce_cnt, all_gather_cnt, reduce_scatter_cnt
+
+
+def test_sequence_parallelism_compilation():
+    temp_dir = tempfile.mkdtemp()
+
+    config = CompilationConfig(
+        level=3,
+        custom_ops=["+rms_norm"],
+        compile_sizes=[4, 8],
+        splitting_ops=[],
+    )
+    config.pass_config.enable_sequence_parallelism = True
+    config.pass_config.dump_graph_dir = Path(temp_dir)
+    config.pass_config.dump_graph_stages = \
```

Comment:
We're actually planning on deprecating the `dump_graph_dir` in favor of depyf inspection, so tests really shouldn't depend on it.